package.json

```diff
@@ -33,7 +33,7 @@
     "content-disposition": "0.5.4",
     "content-type": "~1.0.4",
     "cookie": "0.6.0",
-    "cookie-signature": "1.0.6",
+    "cookie-signature": "1.2.1",
```

Comment:
Can we open it up with `^`? We ahve been doing that as we update these in the v5 branch.

Reply:
Thank you for pointing out the caret.